examples/config-minimal.json

```diff
@@ -9,7 +9,6 @@
         "ReadOnly": false
     },
     "log": {
-        "level": "debug",
-        "output": "/tmp/zot.log"
```

Comment:
Was this intentional?  It seems unrelated.

Reply:
yes, removed now.